android/src/main/kotlin/com/simform/audio_waveforms/AudioPlayer.kt

```diff
@@ -185,6 +179,29 @@ class AudioPlayer(
         }
     }
 
+    fun setReleaseMode(result: MethodChannel.Result, releaseModeType: Int?) {
+        try {
+            releaseModeType?.let {
+                when (releaseModeType) {
+                    0 -> {
+                        this.finishMode = FinishMode.Release
+                    }
+
+                    1 -> {
+                        this.finishMode = FinishMode.Loop
+                    }
+
+                    else -> {
+                        this.finishMode = FinishMode.Pause
+                    }
```

Comment:
We now prefer throwing error when there is unknown mode so,
0 -> stop
1-> loop
2 -> pause
else -> error